src/main/java/com/gachigang/ontherun/config/handler/ExceptionWebHandler.java

```diff
@@ -0,0 +1,32 @@
+package com.gachigang.ontherun.config.handler;
+
+import com.gachigang.ontherun.model.dto.ErrorMessage;
+import jakarta.annotation.Nonnull;
+import lombok.NonNull;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Component
+public class ExceptionWebHandler {
```

Comment:
I believe it should have Helper or Utils postfix (Because Handler should hendle something, in our case it just helps us to do something)

Reply:
Fixed